src/main/java/digital/slovensko/autogram/drivers/PKCS11TokenDriver.java

```diff
@@ -13,7 +12,8 @@ public PKCS11TokenDriver(String name, Path path, boolean needsPassword, String s
     }
 
     @Override
-    public AbstractKeyStoreTokenConnection createTokenWithPassword(Integer slotId, char[] password) {
-        return new Pkcs11SignatureToken(getPath().toString(), new PrefilledPasswordCallback(new KeyStore.PasswordProtection(password)), -1, slotId, null);
+    public AbstractKeyStoreTokenConnection createTokenWithPassword(Integer slotIndex, char[] password) {
```

Comment:
Tu som to asi ja zle prerobil imho to ma byt naozaj slotId (je tam rozdiel v indexovani)

Reply:
Aj doteraz sme používali slotIndex, len sa to tu volalo slotId. Ale slotId je iné ako slotIndex a chceme nechať pôvodnú funkcionalitu, že podľa indexu. Tie slotId sú väčšinou divné a človek sa tam vie zamotať s medzerami a neviem čím - to je vlastne nejaký label slotu.

Reply:
Praveze vychadzam z original super metod z DSS a tam to vyzera takto
```
   public Pkcs11SignatureToken(String pkcs11Path, PasswordInputCallback callback, int slotId, String extraPkcs11Config) {
        this(pkcs11Path, callback, slotId, -1, extraPkcs11Config);
    }
```